analytics/views.py

```diff
@@ -1132,6 +1133,12 @@ def support(request: HttpRequest) -> HttpResponse:
             attach_discount_to_realm(realm, new_discount)
             msg = f"Discount of {realm.string_id} changed to {new_discount} from {current_discount} "
             context["message"] = msg
+        elif request.POST.get("change_subdomain", None) is not None:
+            subdomain = str(request.POST.get("change_subdomain"))
```

Comment:
Why do we need the string conversion here?

Reply:
I realized it is not actually required..